tools/README.md

```diff
@@ -0,0 +1,61 @@
+# Using prebuilt images with gRPC OSS benchmarks
+
+The scripts in this folder:
+* Build the images have pre-built workers embedded.
+* Push the images to specified registry.
+* Delete the images from specified registry.
+
+## Build and push images
+
+The script [prepare_prebuilt_workers.go](pre_built_workers/prepare_prebuilt_workers.go) 
+build images and push them to a user specified registry. 
+The following example build and push prebuilt cxx and go worker images.
+```
+go run ./prebuilt_workers/prepare_prebuilt_workers/prepare_prebuilt_workers.go \
+ -l cxx:master \
+ -l go:master \
+ -p gcr.io/grpc-testing/e2etesting/pre_built_workers \
+ -t user-specified-tag \
+ -r test-infra/container/pre_built_workers/
+```
+
+The built `cxx` and `go` images contains the workers built from commit/branch wish 
```

Comment:
nit: add "we" so it helps the clauses differentiate the subjects. Makes it a little cleaner to consume. 

```suggestion
The built `cxx` and `go` images contains the workers built from the commit/branch we wish 
```